omp/preconditioner/batch_jacobi_kernels.cpp

```diff
@@ -37,7 +37,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 
 #include "core/matrix/batch_struct.hpp"
-#include "omp/matrix/batch_struct.hpp"
+#include "reference/matrix/batch_struct.hpp"
```

Comment:
Maybe if this is common in reference and omp, we can move it to `common/` instead ?

Reply:
But `common` is currently used for cuda and hip. This cannot be used for those, but only on the "host". Maybe we create a new folder called `common_host` or something?

Reply:
I mean you can put it in common and include it only in reference and omp. They dont have to be with the `.inc` extension either. Just a normal `.hpp` should be good.

Reply:
I still think it would be a bit confusing that something in `common` does not work on cuda and hip. Would you be okay with a `common_host` or `common_cpu` folder for this?